staging/src/k8s.io/apiserver/pkg/util/proxy/streamtranslator.go

```diff
@@ -115,10 +120,15 @@ func (h *StreamTranslatorHandler) ServeHTTP(w http.ResponseWriter, req *http.Req
 		//nolint:errcheck   // Ignore writeStatus returned error
 		if statusErr, ok := err.(*apierrors.StatusError); ok {
 			websocketStreams.writeStatus(statusErr)
+			metrics.IncStreamTranslatorRequest(req.Context(), strconv.Itoa(http.StatusInternalServerError))
```

Comment:
is this an internal error or an error from the backend? seems like we want to treat this differently than actual internal errors during spdy client construction above, or errors of unexpected types below

Reply:
Updated to retrieve the status code from the `StatusError` and increment that code.